azerothcore/azerothcore-wotlk
# fix(DB/Event):  Brewfest - Synthebrew Goggles and Ram Racing Reins

## CHANGES PROPOSED:
-  Fix smart_scripts for synthebrew goggles and ram racing reins
-  Fix conditions and gossip_menu_option for synthebrew goggles


## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3495
- Credits: @genion18



## TESTS PERFORMED:
- Apply sql and restart server
- Check that glodrak huntsniper and goldark snipehunter now provides the synthebrew goggles
- Check that driz tumblequick provides the ram racing reins



## HOW TO TEST THE CHANGES:

- Apply sql and restart server
- Check that glodrak huntsniper and goldark snipehunter now provides the synthebrew goggles
- Check that driz tumblequick provides the ram racing reins

## KNOWN ISSUES AND TODO LIST:




## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
